Utils/Dataflow/019_esFormat/esFormat.php

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env php
+<?php
+function exception_error_handler($errno, $errstr, $errfile, $errline ) {
+   throw new ErrorException($errstr, 0, $errno, $errfile, $errline);
+}
+set_error_handler("exception_error_handler");
+
+$DEFAULT_INDEX = 'prodsys';
```

Comment:
I think here must by 'prodsys_rucio', according to updated mapping file

Reply:
`'prodsys'` was left to correspond the original script version :)
But you`re right, it must be changed here (and kept up to date later).